src/test/java/ruslan/shastkiv/bookstore/service/OrderServiceTest.java

```diff
@@ -0,0 +1,183 @@
+    @Mock
+    private OrderRepository orderRepository;
+    @Mock
+    private UserService userService;
+    @Mock
+    private OrderMapper orderMapper;
+    @Mock
+    private OrderItemMapper orderItemMapper;
+    @Mock
+    private OrderItemRepository orderItemRepository;
+
+    @Test
+    @DisplayName("""
+            placeOrderByUserId()
+            - Should place an order by user ID and return OrderDto
+            """)
+    public void placeOrderByUserId_validRequest_returnOrderDto() {
+        User user = createUser(USER_ID);
+        Authentication authentication = getAuthentication(user);
+        ShoppingCart cart = createCart(USER_ID, user, new HashSet<>());
+        CartItem cartItem = createCartItem(FIRST_BOOK_ID, cart);
+        cart.getCartItems().add(cartItem);
+        Order order = createOrder(user, List.of());
+        OrderItem orderItem = createOrderItem(order, FIRST_BOOK_ID);
+        order.getOrderItems().add(orderItem);
+        OrderDto expectedDto = createOrderDto(ORDER_ID_1, List.of(
+                createOrderItemDto(FIRST_BOOK_ID)), Order.Status.PENDING);
+
+        when(userService.getUserId(authentication)).thenReturn(USER_ID);
+        when(shoppingCartService.findShoppingCart(USER_ID)).thenReturn(cart);
+        when(orderRepository.save(any(Order.class))).thenReturn(order);//todo
+        when(orderMapper.toDto(any(Order.class))).thenReturn(expectedDto);
+        when(orderMapper.toOrder(any(ShoppingCart.class))).thenReturn(order);
+        when(orderItemMapper.toOrderItem(cartItem)).thenReturn(orderItem);
+
+        OrderDto actualDto = orderService.placeOrderByUserId(
+                authentication, createPlaceOrderRequestDto(USER_ID));
+
+        verify(shoppingCartService, times(ONE_INVOCATION)).clearShoppingCart(cart);
+        Assertions.assertEquals(expectedDto, actualDto);
+    }
+
+    @Test
+    @DisplayName("""
+            getAllOrdersByUserId()
+            - Should get all orders by user ID
+            """)
+    public void getAllOrdersByUserId_validRequest_returnPageableOrderDto() {
+        OrderItemDto orderItemDto = createOrderItemDto(ORDER_ITEM_ID_1);
+        OrderDto orderDto = createOrderDto(ORDER_ID_1, List.of(orderItemDto), Order.Status.PENDING);
+        Order order = createOrder(createUser(USER_ID), new ArrayList<>());
+        OrderItem orderItem = createOrderItem(order, FIRST_BOOK_ID);
+        order.getOrderItems().add(orderItem);
+        Page<Order> orderPage = new PageImpl<>(List.of(order), PAGEABLE, PAGE_SIZE_1);
+        PageImpl<OrderDto> expectedPage = new PageImpl<>(List.of(orderDto), PAGEABLE, PAGE_SIZE_1);
+
+        when(orderMapper.toDto(any(Order.class))).thenReturn(orderDto);
+        when(orderRepository.findAllByUserId(USER_ID, PAGEABLE)).thenReturn(orderPage);
+        Page<OrderDto> actualPage = orderService.getAllOrdersByUserId(USER_ID, PAGEABLE);
+
+        Assertions.assertEquals(expectedPage, actualPage);
```

Comment:
```suggestion
    public void getAllOrdersByUserId_validRequest_returnPageableOrderDto() {
        // Given
        OrderItemDto orderItemDto = createOrderItemDto(ORDER_ITEM_ID_1);
        OrderDto orderDto = createOrderDto(ORDER_ID_1, List.of(orderItemDto), Order.Status.PENDING);
        Order order = createOrder(createUser(USER_ID), new ArrayList<>());
        OrderItem orderItem = createOrderItem(order, FIRST_BOOK_ID);
        order.getOrderItems().add(orderItem);
        Page<Order> orderPage = new PageImpl<>(List.of(order), PAGEABLE, PAGE_SIZE_1);
        PageImpl<OrderDto> expectedPage = new PageImpl<>(List.of(orderDto), PAGEABLE, PAGE_SIZE_1);

        when(orderMapper.toDto(any(Order.class))).thenReturn(orderDto);
        when(orderRepository.findAllByUserId(USER_ID, PAGEABLE)).thenReturn(orderPage);
        
        // When
        Page<OrderDto> actualPage = orderService.getAllOrdersByUserId(USER_ID, PAGEABLE);
        
        // Then
        Assertions.assertEquals(expectedPage, actualPage);
```
You can increase readability in this way